src/App.vue

```diff
@@ -29,10 +35,27 @@ import 'reveal.js/dist/reveal.css'
 import 'reveal.js/plugin/highlight/monokai.css'
 import 'reveal.js/dist/theme/white.css'
 
-const dataSeparator = '\r?\n---\r?\n'
-const dataSeparatorVertical = '\r?\n--\r?\n'
+import { getMediaMimeTypes } from './helpers/mediaMimeTypes'
+import { id as appId } from '../public/manifest.json'
+
 const themeStore = useThemeStore()
+const { loadFolderForFileContext, currentFileContext, activeFiles } = useAppDefaults({
+  applicationId: appId
+})
+const { getUrlForResource, revokeUrl } = useAppFileHandling({
+  clientService: useClientService()
+})
+const appsStore = useAppsStore()
```

Comment:
![image](https://github.com/JankariTech/web-app-presentation-viewer/assets/8559121/93b1b658-58c7-4fbe-8e86-d7a9f169c3fb)

Cannot load the presentation

Reply:
hmm, is `useAppsStore` only available on web-master?
Let me check

Reply:
Yeah, the code is not available in web-8.0. :cry:

Reply:
This PR will be merged to the main branch which will be v2.0.0.
We have release [v1.0.0](https://github.com/JankariTech/web-app-presentation-viewer/releases/tag/1.0.0)